### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by wangwenqi567** [RapidAI/RapidTable#21] [Ideas]
我想问一下，表格识别RapidTable的代码里是指定使用CPU的，但是这可能导致CPU突然特别高，能否改为指定GPU去运行，在哪里去指定呢，谢谢

**Comment by Joker1212** [comment]
emmm,理论上不会造成cpu突然特别高的吧，因为模型本身很小，另外如果需要gpu的话，可以使用paddleX的版本的模型，不过有需求的话，我下个版本增加一个onnx的gpu版本吧，给大家多一点选择

**Reply by wangwenqi567** [reply]
跑了一下，CPU使用率会上2000，20个核，有些高，好的，谢谢

**Reply by Joker1212** [reply]
onnx会这样吗？emmm,没有使用paddle-ocr吧？那个好像内存泄漏

**Reply by wangwenqi567** [reply]
onnx好像都会这样，短暂的导致CPU利用率极高，之前用过Cnocr的ch_PP-OCRv3_det的onnx版本，也会导致这种情况，不过看了其他讨论，将rapidocr-runtime换成了GPU版的rapidocr_paddle，CPU的使用率下降了一半，速度也变快了。

**Reply by SWHL** [reply]
因为大部分算力都跑GPU上去了，CPU这里自然少一些

**Reply by wangwenqi567** [reply]
是的，不过还是会到12核，虽然时间非常短，所以如果下个版本有GPU版本的就好了，感谢

**Reply by SWHL** [reply]
这个可以有的，因为输入shape是固定的，完全可以使用onnxruntime-gpu来推理的

**Comment by Joker1212** [comment]
0.3.0版本已经支持
